### PR TITLE
Updated Viber.Viber to v15.5.0.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
Closes #17786

- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17694)

### CODE REVIEW

File: manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
+# Created using YamlCreate.ps1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
 PackageIdentifier: Viber.Viber
-PackageVersion: 15.4.1.1
+PackageVersion: 15.5.0.0
 PackageLocale: en-US
-Publisher: Viber Media S.a.r.l
+Publisher: 2010-2021 Viber Media S.a.r.l
```
Comment (ItzLevvie) [inline manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml:7]:
```suggestion
Publisher: Viber Media S.a.r.l
```
It shows up for me as:
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml:7]: wat, is it the MSI or EXE? I got suprised when i noticed the exe had `2010-2021 Viber Media S.a.r.l` as publisher

Comment (ItzLevvie) [reply manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml:7]: It was the `.msi` installer which winget picked as default when I installed it.

Comment (OfficialEsco) [reply manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml:7]: The MSI uses the ProductCode to match so its okay, or we need split them again

Comment (OfficialEsco) [reply manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml]: There might actually be a issue where x86 is being picked over neutral, it installed the exe once i swapped the exe to x86

```
Found Viber [Viber.Viber]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.cdn.viber.com/desktop/windows/ViberSetup.exe
  ██████████████████████████████   102 MB /  102 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

image

However it somehow did find a match
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget list
Name                  Id                                           Version          Available   Source
------------------------------------------------------------------------------------------------------
Viber                 Viber.Viber                                  15.5.0.0
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**